.github/workflows/default.yml

```diff
@@ -30,3 +30,5 @@ jobs:
 
       - name: run tests
         run: make test
+        env:
+          PLENARY_DIR: vendor/plenary.nvim
```

Comment:
Why would you need another folder for downloading plenary.nvim? any specific usecase?

Reply:
Basically me and @cuducos has done this on ruby.nvim for don't set vendor on gitignore.
We set this on GitHub actions because GitHub loses yourself when folder is /tmp

Reply:
References:
https://github.com/vinibispo/ruby.nvim/pull/16